core/src/test/resources/AppConstants.properties

```diff
@@ -1 +1,4 @@
 SPRING.CONFIG.LOCATIONS=springContext.xml
+
+transactionmanager.log.dir=target
+transactionmanager.uid=unique
```

Comment:
Deze naam kan, maar een iets beschrijvender gaat in een later stadium wel helpen denk ik.